src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/MsQuicEnums.cs

```diff
@@ -0,0 +1,204 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Net.Quic.Implementations.MsQuic.Internal
+{
+    internal enum QUIC_EXECUTION_PROFILE : uint
```

Comment:
I'm really NOT the style guy but in some other places we made the names C# friendly.

https://github.com/dotnet/runtime/blob/79ae74f5ca5c8a6fe3a48935e85bd7374959c570/src/libraries/Common/src/System/Net/SecurityStatusPal.cs#L25-L30

and we use int unless uint is really needed (even if underlying C is uint)

Reply:
I won't change the naming scheme in this PR, that's too revolutionary :smiley:.
We might change it in a separate one or we might go a different direction if we chose to generate the interop.